examples/cartesian_reconstruction.py

```diff
@@ -0,0 +1,98 @@
+"""
+Neuroimaging cartesian reconstruction
+=====================================
+
+Author: Chaithya G R
+
+In this tutorial we will reconstruct an MRI image from the sparse kspace
+measurments.
+
+Import neuroimaging data
+------------------------
+
+We use the toy datasets available in pysap, more specifically a 2D brain slice
+and the acquisition cartesian scheme.
+"""
+
+# Package import
+from modopt.math.metrics import ssim
+from mri.numerics.reconstruct import sparse_rec_fista
+from mri.numerics.fourier import FFT2
+from mri.numerics.utils import generate_operators
+from mri.numerics.utils import convert_mask_to_locations
+import pysap
+
+# Third party import
+import numpy as np
+
+# Loading input data
+image = pysap.utils.load_image('../../../Data/Pysap_examples/base_image.npy')
+
+# Obtain K-Space Cartesian Mask
+mask = pysap.utils.load_image(
+    '../../../Data/Pysap_examples/mask_cartesian.npy')
+
+# View Input
+image.show()
+mask.show()
+
+#############################################################################
+# Generate the kspace
+# -------------------
+#
+# From the 2D brain slice and the acquisition mask, we generate the acquisition
+# measurements, the observed kspace.
+# We then reconstruct the zero order solution.
+
+
+# Get the locations of the kspace samples
+kspace_loc = convert_mask_to_locations(np.fft.fftshift(mask.data))
+# Generate the subsampled kspace
+fourier_op = FFT2(samples=kspace_loc, shape=image.shape)
+kspace_data = fourier_op.op(image)
+
+# Zero order solution
+image_rec0 = pysap.Image(data=fourier_op.adj_op(kspace_data),
+                         metadata=image.metadata)
+image_rec0.show()
+
+# Calculate SSIM
+base_ssim = ssim(image_rec0, image)
+print(base_ssim)
+
+#############################################################################
+# FISTA optimization
+# ------------------
+#
+# We now want to refine the zero order solution using a FISTA optimization.
+# Here no cost function is set, and the optimization will reach the
+# maximum number of iterations.
+
+# Generate operators
+gradient_op, linear_op, prox_op, cost_op = generate_operators(
+    data=kspace_data,
+    wavelet_name="sym8",
+    samples=kspace_loc,
+    nb_scales=4,
+    mu=2.364e-07,
```

Comment:
would `2 * 1e-7` work? It's way more readable and concise.
I mean would we still be better than zero-filled recon? If yes then use this and explain how you got to this value in a comment (i.e. grid search).

Reply:
Well it surely better, just that this is exactly where I received the maxima in the grid search. 
Yes, I will document on how we got this value. Maybe I will initialize this as a parameter above and add these comments.

However, I think it is better to use the right value of `mu` than a readable value of `mu`.

Reply:
I don't think so. This "right value" of `mu` was grid searched but it doesn't mean that it's the "right one". It wouldn't work in other situations, with a different noise, so it's better to just be readable, explain how and not have people wonder why there is such an exact value here.

Reply:
I mean from viewpoint of results, we would want it to be as good as possible. 
With this in mind, I meant the `right_value` as the value that gives the best reconstruction. 
I am fine to make it readable if the reconstruction results and SSIM is not way off.

Reply:
They shouldn't be with `2 * 1e-7`

Reply:
Are you suggesting to have codes with 2 * 1e-7 but the results with 2.364e-07 ? 
I mean I dont know how much would the SSIM change, but would it not kill re-producibility?

Reply:
I am suggesting to have codes and results with `2 * 1e-7`

Reply:
I checked that SSIM is 0.9143 (baseline is 0.82) 
I think we can hold on to 2 * 1e-7 this. Resolving